scintillometry/tests/data/B1937+21_NANOGrav_11yv1.gls.par

```diff
@@ -0,0 +1,1059 @@
+PSR              B1937+21
```

Comment:
Why do we need two par files? Can we just stick with the much shorter NGC-* one?